content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -1,634 +0,0 @@
----
```

Comment:
If we are removing this file, we'll likely want to create a redirect to another page so we don't break bookmarks.

Reply:
https://github.com/kubernetes/website/blob/master/static/_redirects
(but I'm not sure where we should point it now...)

Reply:
Good idea, I've updated that [here](https://github.com/kubernetes/website/pull/26682/commits/8135b58f3010cb5a0f9e0f0acbb8586c83b2a51f).